### PR TITLE
MESH-2151/change-compliance-rules

### PR DESCRIPTION
**Pull request by HenriqueNogara**
### new features

- Change compliance rules - if there are no rules returns `Ok`.

### CODE REVIEW

**Review by Neopallium — APPROVED** [review]

**Comment by Neopallium** [comment]
I edited the description.  We should include in the release notes about the changed logic.

**Review by adamdossa — APPROVED** [review]

**Comment by adamdossa** [comment]
Approved, but marked as WIP whilst we wait for final product approval and decide if this constitutes a breaking change.

**Review by adamdossa — APPROVED** [review]

**Pull request merged**
